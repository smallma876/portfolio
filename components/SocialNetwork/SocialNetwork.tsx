import Image from "next/image";
import Link from "next/link";
import React from "react";
import { Box, Flexbox } from "../../shared/components/Flexbox/Flexbox";

const socialNetworks = [
  {
    src: "./images/social/linkedin.svg",
    alt: "linkedin",
    link: "https://www.linkedin.com/in/sergio-ruben-mallma-chavez-5b2b0913b",
  },
  {
    src: "./images/social/instagram.svg",
    alt: "instagram",
    link: "https://www.instagram.com/sergio.mdc/",
  },
  {
    src: "./images/social/facebook.svg",
    alt: "facebook",
    link: "https://web.facebook.com/sergio.mallma.92",
  },
];

const SocialNetwork = () => {
  return (
    <div>
      <Flexbox gap="0.5rem">
        {socialNetworks.map((social) => (
          <Box key={social.alt}>
            <Link href={social.link} target="_blank">
              <Image
                src={social.src}
                alt={social.alt}
                width={20}
                height={20}
                style={{ cursor: " pointer" }}
              />
            </Link>
          </Box>
        ))}
      </Flexbox>
    </div>
  );
};

export default SocialNetwork;
