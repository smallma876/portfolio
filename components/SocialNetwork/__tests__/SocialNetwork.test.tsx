import { render, screen } from "@testing-library/react";
import SocialNetwork from "../SocialNetwork";

const renderComponent = async () => {
  const { container } = await render(<SocialNetwork />);

  return container;
};

describe("SocialNetwork", () => {
  it("should render match snapshot", () => {
    const container = renderComponent();
    expect(container).toMatchSnapshot();
  });
});
