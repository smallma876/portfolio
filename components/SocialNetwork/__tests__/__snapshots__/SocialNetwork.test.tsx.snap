// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SocialNetwork should render match snapshot 1`] = `Promise {}`;
